tests/test_eigendistortion.py

```diff
@@ -139,17 +139,16 @@ def test_display(self, model, einstein_img, color_img, method, k):
         elif method == "randomized_svd":  # svd only has top k not bottom k eigendists
             with pytest.raises(AssertionError):
                 eigendist.plot_distorted_image(eigen_index=-1)
-                   
+        plt.close("all")    
 
 class TestAutodiffFunctions:
 
     @pytest.fixture(scope='class')
     def state(self, einstein_img):
         """variables to be reused across tests in this class"""
-        torch.manual_seed(0)
 
         k = 2  # num vectors with which to compute vjp, jvp, Fv
-        einstein_img = einstein_img[..., :16, :16]  # reduce image size
+        einstein_img = einstein_img[..., 100:100+16, 100:100+16]  # reduce image size
```

Comment:
why grab a different part of the image?

Reply:
There wasn't much going on in the top corner of the image. Wanted to use a patch with more variation so that the eigenvectors would be better behaved.